plugins/participants.js

```diff
@@ -0,0 +1,167 @@
+module.exports = (controller) => {
+    const renderParticipantsBlocks = async (participants) => {
+        return Promise.all(
```

Comment:
since `renderParticipantBlocks` isn't actually asynchronous, we probably don't need `Promise.all` here.